utils.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2020 and later: Google, Inc.
+
+import os
+import random
+import shutil
+
+def _check_create_dir(dir_path):
```

Comment:
No need to write this function, just use 
os.makedirs(name, exist_ok=True)
instead
see https://docs.python.org/3/library/os.html